docs/client/Account.md

```diff
@@ -0,0 +1,15 @@
+# Account
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**Description** | **string** |  | [optional] [default to null]
+**Href** | **string** |  | [optional] [default to null]
+**Id** | **string** |  | [optional] [default to null]
+**Name** | **string** |  | [default to null]
+**Parent** | [***Link**](Link.md) |  | [optional] [default to null]
```

Comment:
Can you check into the "null" vs "nil" that is showing up in the docs?

Reply:
It looks like Swagger-Codegen is include `null` here since we are not defining a default value in the Swagger schema. https://swagger.io/specification/v2/#parameter-object

It's most likely using `null` because codegen is written in Java.
Not sure there is anything we can do except to run a post processor to replace these.

Reply:
https://pureport.atlassian.net/browse/PE-1044